analyzers/its/expected/Nancy/Nancy--net452-S3900.json

```diff
@@ -1692,19 +1692,6 @@
 },
 {
 "id":  "S3900",
-"message":  "Refactor this method to add validation of parameter 'e' before using it.",
-"location":  {
-"uri":  "sources\Nancy\src\Nancy\Helpers\HttpUtility.cs",
-"region":  {
-"startLine":  192,
```

Comment:
TN (reassign in line 148)
https://github.com/SonarSource/sonar-dotnet/blob/e641f52adc52e9b8743ebb3a98d0fb6582360bab/analyzers/its/sources/Nancy/src/Nancy/Helpers/HttpUtility.cs#L192